local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -730,6 +732,64 @@ void LocalPlannerNode::distanceSensorCallback(
     publishGround();
   }
 }
+
+void LocalPlannerNode::px4ParamsCallback(const mavros_msgs::Param& msg) {
+  // collect all px4 parameters needed for model based trajectory planning
```

Comment:
These comments should go in the header file. Try to use similar style to [here](https://github.com/PX4/avoidance/blob/9b3d5ed6b2b9bace15f60a0b24eec5a8b3a7da7a/local_planner/src/nodes/common.h#L15)